charts/loki-simple-scalable/templates/read/statefulset-read.yaml

```diff
@@ -13,6 +13,9 @@ spec:
       partition: 0
   serviceName: {{ include "loki.readFullname" . }}-headless
   revisionHistoryLimit: {{ .Values.loki.revisionHistoryLimit }}
+  persistentVolumeClaimRetentionPolicy:
+    whenDeleted: Delete
+    whenScaled: Delete
```

Comment:
Forgive my ignorance - which parts of the `read` target are using statefulsets? The compactor and ruler?

Reply:
it's actually probably less important for the `compactor` and `ruler`, and most important for fetched indexes, which was previously the job of the `querier`, but is now the job of the `indexgateway` which is running on the `read` nodes as a result of a recent PR to loki.

Reply:
@dannykopping added a comment, and removed the change to write pods since, as @pracucci pointed out, better to play it safe with that data as it's not replaceable like read path data.